pygotham/users/models.py

```diff
@@ -20,6 +20,8 @@
     db.Column('role_id', db.Integer, db.ForeignKey('roles.id')),
 )
 
+user_talks = db.Table('speakers', Speaker.metadata, autoload=True)
```

Comment:
Also, is this table object even necessary? Can't `Speaker` be used as the value of `secondary` in the relationship between `User` and `Talk`?